app/Resources/views/page.html.twig

```diff
@@ -212,7 +212,11 @@
         </noscript>
     {% endif %}
 
-    <div class="global-wrapper" data-behaviour="FragmentHandler Math">
+    <div class="global-wrapper" data-behaviour="FragmentHandler Math"
+        {% if item is defined and item is not null %}
+            data-article-type="{% if item.type is defined %}{{ model_name(item.type) }}{% else %}{{ model_name(item.identifier.type) }}{% endif %}"
```

Comment:
`model_name()` gets the human-readable form from an ID. We just want the ID here so can lose the call.

Reply:
Currently this will be for all item types not just articles, so `data-article-type` should be `data-item-type`.